controllers/metal3.io/bmceventsubscription_controller.go

```diff
@@ -88,7 +87,7 @@ func (r *BMCEventSubscriptionReconciler) Reconcile(ctx context.Context, request
 		if k8serrors.IsNotFound(err) {
 			reqLogger.Error(err, "baremetalhost not found", "host", subscription.Spec.HostName)
 
-			message := fmt.Sprintf("baremetal host \"%s\"", subscription.Status.Error)
+			message := fmt.Sprintf("baremetal host \"%q\"", subscription.Status.Error)
```

Comment:
The `%q` already results in a quoted value so we do not need the extra double quotes.
```suggestion
			message := fmt.Sprintf("baremetal host %q", subscription.Status.Error)
```

Reply:
done